[IMP] contributing: refresh doc for new layout and content structure (#1024)
odoo/documentation: content/contributing/documentation/content_guidelines.rst

   - :ref:`RST cheat sheet: headings <contributing/headings>`
   - :ref:`RST cheat sheet: specialized directives <contributing/specialized-directives>`

.. _contributing/organizing-documentation:

Organizing the documentation
============================

> When writing documentation about a given topic, try to keep related pages organized.

Reviewer: I knew what you meant with "related pages" because I knew the topic already. It could be interpreted differently, such as "related topics," rather than the files that are in the same folder.

Author: I inspired myself from the note on the top of the page:
> the other documents related to this section of the documentation

Do you want to change this formulation too?

Reviewer: That's something you modified in this PR ^^ It wasn't written like that before.
The meaning is different though, to me.

"the other documents related to this section of the documentation."
--> "related" works here since the topics are related in meaning.

"When writing documentation about a given topic, try to keep related pages organized."
--> I want to understand "related" from a "topic" understanding, especially since it's written a few words before. 

I think it's all good now :)

> For most topics, a single page will do the job. Place it in the appropriate section of the
> documentation (e.g., content related to the CRM app go under :menuselection:`Applications
> -> Sales -> CRM`) and follow the :ref:`document structure <contributing/document-structure>`
> guidelines.

Reviewer: - avoid the future tense. "a single page does the job" or "should do the job"
- 
```suggestion
For most topics, a single page should do the job. Place it in the appropriate section of the
documentation (e.g., content related to the CRM app go under :menuselection:`Applications
-> Sales -> CRM`) and follow the :ref:`document structure <contributing/document-structure>`
guidelines.
```

Author: I thought about this but didn't realize that "will do" was also valid in the present form.

> | For a more complex topic, several pages—each discussing one of its aspects—may be needed.

Reviewer: I would avoid the | to go to the line. This is more "French," while in English, different thoughts are separated into paragraphs.

I advise using it mostly to enhance the "layout" of your content.

```suggestion
For more complex topics, you may need several pages to cover all their aspects.
```
